geomesa-spark/geomesa-spark-accumulo/src/test/scala/org/locationtech/geomesa/spark/accumulo/SparkSQLTestUtils.scala

```diff
@@ -0,0 +1,86 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.spark.accumulo
+
+import java.io.File
+import java.nio.file.Files
+import java.util.{Map => JMap}
+
+import com.vividsolutions.jts.geom.Coordinate
+import org.apache.accumulo.minicluster.{MiniAccumuloConfig, MiniAccumuloCluster}
+import org.geotools.data.simple.SimpleFeatureStore
+import org.geotools.data.{DataUtilities, DataStoreFinder}
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.joda.time.format.ISODateTimeFormat
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreParams}
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.tools.ingest.ConverterIngest
+import org.locationtech.geomesa.utils.geotools.{GeneralShapefileIngest, SimpleFeatureTypes}
+
+import scala.collection.JavaConversions._
+
+object SparkSQLTestUtils {
+  def setupMiniAccumulo(): MiniAccumuloCluster = {
+    val randomDir = Files.createTempDirectory("mac").toFile
+    val config = new MiniAccumuloConfig(randomDir, "password").setJDWPEnabled(true)
+    val mac = new MiniAccumuloCluster(config)
+    mac.start()
```

Comment:
we should probably shut down the cluster after the tests? Along those lines, it would probably be a good idea to have a globally shared mini cluster that is spun up before running any tests and stopped at the end. I believe that is doable through maven - possibly you could add some hooks so that if the tests are run outside maven they will start up a new cluster.